charon/src/pretty/fmt_with_ctx.rs

```diff
@@ -1014,6 +1014,7 @@ impl llbc::Statement {
                     tab
                 )
             }
+            RawStatement::Error => format!("{tab}panic"),
```

Comment:
`Error` isn't a runtime panic in the emitted code, it's a compile-time error in Charon. I think we can indicate that like:
```suggestion
            RawStatement::Error => format!("{tab}/* charon error */"),
```